### PR TITLE
fix: inform user to generate an API key before debugging the API ME/API Plugin with API Key auth templates

### PR DESCRIPTION
**Pull request by huimiu**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/28568450

image

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -250,6 +250,9 @@
   "teamstoolkit.handlers.provisionDescription": "[%s] is successfully created at [local address](%s). Continue to provision and then you can preview the app.",
   "teamstoolkit.handlers.provisionDescription.fallback": "[%s] is successfully created at %s. Continue to provision and then you can preview the app.",
   "teamstoolkit.handlers.provisionTitle": "Provision",
+  "teamstoolkit.handlers.manualStepRequired": "[%s] is successfully created at [local address](%s). Before start debugging, please follow the manual steps outlined in the README file.",
```
Comment (supkasar) [inline packages/vscode-extension/package.nls.json:253]: Proposed: [%s] is created at [local address](%s). Before debugging, follow the steps in the README file.

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -250,6 +250,9 @@
...
+  "teamstoolkit.handlers.manualStepRequired.fallback": "[%s] is successfully created at %s. Before start debugging, please follow the manual steps outlined in the README file.",
```
Comment (supkasar) [inline packages/vscode-extension/package.nls.json:254]: Same here.

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -250,6 +250,9 @@
...
+  "teamstoolkit.handlers.manualStepRequiredTitle": "Read more",
```
Comment (supkasar) [inline packages/vscode-extension/package.nls.json:255]: Proposed: Get more info

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (huimiu) [reply packages/vscode-extension/package.nls.json]: Updated, thanks!

File: packages/vscode-extension/package.nls.json (hunk 2 above)
Comment (huimiu) [reply packages/vscode-extension/package.nls.json]: Updated

**Review by supkasar — DISMISSED** [review]

**Review by yiqing-zhao — APPROVED** [review]

**Comment by huimiu** [comment]
Hey @supkasar, could you please help review this again? I made some changes to the string based on Junjie's comments.

**Review by supkasar — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Comment by huimiu** [comment]
Update the screenshot with the latest string content
image

**Pull request merged**